acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/UpCheckAcceptanceTest.java

```diff
@@ -68,6 +102,15 @@ public void deleteRequestMustTimeout() {
         ethSigner().httpRequests().deleteExceptingTimeout(UP_CHECK_PATH);
 
     assertThat(reply).isNotNull();
-    assertThat(reply.getMessage()).isEqualTo(TIMEOUT_MESSAGE);
+    verifyOkHttpResponse(reply);
+  }
+
+  private void verifyOkHttpResponse(final SocketTimeoutException reply) {
+    // OkHttp appears to have a race condition whereby the thrown exception can have either
+    // of 2 possible messages - it is non-deterministic.
+    assertThat(
```

Comment:
nit: think you can use isIn here to make this simpler
assertThat(reply.getMessage()).isIn(TIMEOUT_MESSAGE, READ_TIMED_OUT_MESSAGE);